ray-operator/controllers/ray/common/pod.go

```diff
@@ -417,6 +418,16 @@ func concatenateContainerCommand(nodeType rayiov1alpha1.RayNodeType, rayStartPar
 		}
 	}
 
+	if _, ok := rayStartParams["dashboard-agent-listen-port"]; !ok {
+		rayStartParams["dashboard-agent-listen-port"] = fmt.Sprint(DefaultDashboardAgentListenPort)
+		for _, port := range ports {
+			if port.Name == "dashboard-agent" {
```

Comment:
Hmm, the UI is that the functionality is turned on/off by presence of a port named "dashboard-agent".
This feels a little awkward. Should we just auto-configure the port with a default?

Reply:
For this pr, let's assume it defined in RayService yaml. I can improve it in the follow up pr.